frontend/package.json

```diff
@@ -29,24 +29,30 @@
     "ts:check": "tsc --noEmit"
   },
   "dependencies": {
-    "axios": "^1.8.2",
     "@newrelic/next": "^0.10.0",
     "@next/third-parties": "^15.1.4",
     "@opentelemetry/api": "^1.8.0",
+    "@rjsf/core": "^5.24.8",
```

Comment:
looks like some of this is unused. Can we remove  rjsf/core, rjsf/validator-ajv8 and react-jsonschema-form?